attributes/default.rb

```diff
@@ -90,14 +90,26 @@
 
 # NVIDIA
 default['cfncluster']['nvidia']['enabled'] = 'no'
-default['cfncluster']['nvidia']['driver_version'] = '450.51.05'
-default['cfncluster']['nvidia']['driver_url'] = 'https://us.download.nvidia.com/tesla/450.51.05/NVIDIA-Linux-x86_64-450.51.05.run'
+default['cfncluster']['nvidia']['driver_version'] = '450.80.02'
+default['cfncluster']['nvidia']['driver_url'] = 'https://us.download.nvidia.com/tesla/450.80.02/NVIDIA-Linux-x86_64-450.80.02.run'
 default['cfncluster']['nvidia']['cuda_version'] = '11.0'
 default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.download.nvidia.com/compute/cuda/11.0.2/local_installers/cuda_11.0.2_450.51.05_linux.run'
 
+# NVIDIA fabric-manager
+default['cfncluster']['nvidia']['fabricmanager']['package'] = "nvidia-fabricmanager-450"
+default['cfncluster']['nvidia']['fabricmanager']['repository_key'] = "7fa2af80.pub"
+default['cfncluster']['nvidia']['fabricmanager_version'] = node['cfncluster']['nvidia']['driver_version']
```

Comment:
let's use the same convention for all the parameters, so: 
`default['cfncluster']['nvidia']['fabricmanager_version']` --> `default['cfncluster']['nvidia']['fabricmanager']['version']`